internal/pipeline/testdata/packagewithkrmignore/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: app
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/set-namespace
+    configMap:
+      namespace: staging
+  - image: gcr.io/kpt-fn/set-label
+    configMap:
+      tier: backend
```

Comment:
Just a reminder, currently set-label function doesn't support this input from `configMap` field. Will fix it soon. https://github.com/GoogleContainerTools/kpt/issues/1487

Reply:
thanks!